playbooks/azure/openshift-cluster/tasks/provision_instance.yml

```diff
@@ -21,7 +21,7 @@
     address_prefix: 192.168.0.0/24
 
 - name: calculate input image
-  command: az image list -g "{{ image_resource_group }}" --query "[?starts_with(name, '{{ image_prefix }}-') && tags.valid=='true'] | sort_by(@, &name) | [-1]"
+  command: az image list -g "{{ openshift_azure_input_image_ns }}" --query "[?starts_with(name, '{{ openshift_azure_input_image_prefix }}-') && tags.valid=='true'] | sort_by(@, &name) | [-1]"
```

Comment:
I think that given we've removed openshift_azure_input_image_prefix from everywhere else and from the documentation, it should be removed from here as well.

Reply:
Sorry, this is my bad, I've fixed the doc now. Removing this would increase the complexity of image building jobs without any value.. I've tried it, it just didn't make sense.